docs/source/models/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -1023,21 +1023,24 @@ Code is below for reference
     - Adding Correlation 
     - No new issues found with adding correlation. `Causes with correlation <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Cause_VV_correlation_07.25.2023.ipynb>`_ and `risks with correlation <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/risks_VV_07.25.2023.ipynb>`_ were checked in the same manner as previous runs. Additionally, `simulant exposure values <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_correlation.ipynb>`_ were checked for correlation in the interactive sim both on initialization and after 25 time steps. It was found that correlation varied around the expected input values. 
   * - 17.0
-    - Add New Observer 
-    - 
+    - Add New Observer - Binned Exposure  
+    - The new observer seems to be working well, V&V for this observer only can be found in the `binned exposure notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/binned_risks_08.08.2023.ipynb>`_. 
```

Comment:
![image](https://github.com/ihmeuw/vivarium_research/assets/51236/cc2e0db0-4413-484b-96ae-4db8408672a8)

Is this also stratified by year?  It would be cool to see how the different scenarios change the distribution over time,

Reply:
Yep - we do have it by year! I can make a by year example. Based on the other data we have, my guess is that we would see a big change in 2023/2024 when interventions scale up and then see them stabilize from there. But I will make some plots.

Reply:
Here is baseline vs polypill over the years: 
![image](https://github.com/ihmeuw/vivarium_research/assets/108029462/95e1a17a-929f-45ef-b823-583f1480d6fe)
![image](https://github.com/ihmeuw/vivarium_research/assets/108029462/5db89d58-3714-4568-9f5d-99f7bbccf083)